plugins/katello/3.15/user_guide/remote_databases/index.md

```diff
@@ -24,6 +24,14 @@ There are two ways to deploy Foreman with remote databases:
     * restore the DBs on remote servers
     * run foreman-installer with right parameters pointing to the databases. It re-configures the databases and start all the services with new DB locations
 
+### Important note:
+
+Pulpcore installs a separate PostgreSQL database and lacks support for external databases. This support is introduced in Katello 3.16.
+
+Foreman and Pulpcore require PostgreSQL 10 or newer. Databases should be upgraded to avoid conflicts. It is recommended to update to version 12.
+
+Note that [Foreman v2.0 requires PostgreSQL v10+](https://theforeman.org/manuals/2.0/index.html#Headlinefeatures).
```

Comment:
You can drop this since it's duplicate from the line above. Perhaps you could add the link there.